content/influxdb/cloud-dedicated/write-data/use-telegraf/configure/manual-config.md

```diff
@@ -0,0 +1,180 @@
+3.  Copy and paste the example configuration into your Telegraf configuration file
+    (typically named `telegraf.conf`).
+
+### Enable and configure the InfluxDB v2 output plugin
+
+To send data to an InfluxDB {{< current-version >}} instance, enable the
+[`influxdb_v2` output plugin](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/influxdb_v2/README.md)
+in the `telegraf.conf`.
+
+To find an example InfluxDB v2 output plugin configuration in the UI:
+
+1. In the navigation menu on the left, select **Load Data** > **Telegraf**.
+
+    {{< nav-icon "load data" >}}
+
+2. Click **InfluxDB Output Plugin**.
+3. Click **Copy to Clipboard** to copy the example configuration or **Download Config** to save a copy.
+4. Paste the example configuration into your `telegraf.conf` and specify the options below.
+
+The InfluxDB output plugin configuration contains the following options:
+
+##### urls
+
+An array of URLs for your InfluxDB {{< current-version >}} instances.
+To write to InfluxDB Cloud Dedicated, include your InfluxDB Cloud Dedicated cluster URL using the HTTPS protocol:
+
+      ```toml
+      ["https://cluster-id.influxdb.io"]
+      ```
+
+##### token
+
+Your InfluxDB Cloud Dedicated [database token](/influxdb/cloud-dedicated/admin/tokens/) with _write_ permission to the database.
+For information about viewing tokens, see [List tokens](/influxdb/cloud-dedicated/admin/tokens/list/).
+
+{{% note %}}
+###### Avoid storing tokens in `telegraf.conf`
+
+We recommend storing your tokens by setting the `INFLUX_TOKEN` environment
+variable and including the environment variable in your configuration file.
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[macOS or Linux](#)
+[Windows](#)
+{{% /tabs %}}
+
+{{% tab-content %}}
+```sh
+export INFLUX_TOKEN=YourAuthenticationToken
+```
+{{% /tab-content %}}
+
+{{% tab-content %}}
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[PowerShell](#)
+[CMD](#)
+{{% /code-tabs %}}
+
+{{% code-tab-content %}}
+```sh
+$env:INFLUX_TOKEN = "YourAuthenticationToken"
+```
+{{% /code-tab-content %}}
+
+{{% code-tab-content %}}
+```sh
+set INFLUX_TOKEN=YourAuthenticationToken
+# Make sure to include a space character at the end of this command.
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+_See the [example `telegraf.conf` below](#example-influxdb_v2-configuration)._
+{{% /note %}}
+
+##### organization
+
+For InfluxDB Cloud Dedicated, set this to an empty string (`""`).
+
+##### bucket
+
+The name of the InfluxDB Cloud Dedicated database to write data to.
+
+#### Example influxdb_v2 configuration
+
+The following example shows a minimal [`outputs.influxdb_v2`](/{{< latest "telegraf" >}}/plugins/#output-influxdb_v2) configuration for writing data to InfluxDB Cloud Dedicated:
+
+```toml
+[[outputs.influxdb_v2]]
+  urls = ["https://cluster-id.influxdb.io"]
+  token = "${INFLUX_TOKEN}"
+  organization = ""
+  bucket = "DATABASE_NAME"
+```
```

Comment:
Either get rid of this example or my suggestion above.

~~~suggestion
#### Example influxdb_v2 configuration

The following example shows a minimal [`outputs.influxdb_v2`](/{{< latest "telegraf" >}}/plugins/#output-influxdb_v2) configuration for writing data to InfluxDB Cloud Dedicated:

```toml
[[outputs.influxdb_v2]]
  urls = ["https://cluster-id.influxdb.io"]
  token = "${INFLUX_TOKEN}"
  organization = ""
  bucket = "DATABASE_NAME"
```
~~~